core/controllers/editor.py

```diff
@@ -294,6 +309,8 @@ def _get_exploration_data(self, exploration_id, version=None):
             'tags': exploration.tags,
             'title': exploration.title,
             'version': exploration.version,
+            'is_version_of_draft_valid': is_version_of_draft_valid,
+            'draft_changes': draft_changes
```

Comment:
@jacobdavis11 `draft_changes` (assuming that it refers to lost changes) will be used only in case when `is_version_of_draft_valid` is `False` ? 
Also, the doc mentions about initialising the frontend changeList to the one which is received from backend. If that's still the case, there are a couple of things:
- What is the purpose of initialing the changeList on the frontend ?
- Where is the changeList being sent from the backend ?